### PR TITLE
Feature/90 make complaint

### LINKED ISSUE #90: Make Complaint
### Describe the solution you'd like
Update profile dropdown
Display complaint form. (Image from [https://icons8.com/icon/63238/icon])
Validate input. 
Save details in DB. 

### UI example
Change 'My Complaints' as Complaint
image

image

### PR DESCRIPTION
**Pull request by amanasamsudeen**
## Purpose

The purpose of this PR is to fix #90 

### Preview Link

https://pr-{PR_NUMBER}-petso.surge.sh/

##  Checklist
- [ ] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: app/controllers/Complaints.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    }
+
+               
+            }
+
+
+
+            $this->view('users/user/inquries', $data);
+        }
+    }
```
Comment (pasindur99) [inline app/controllers/Complaints.php]: EOL

File: app/controllers/Projects.php [hunk 2]
```diff
@@ -543,6 +543,10 @@ public function viewAllProjects() {
     }
 
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+>>>>>>> 8374cb0 (volunteer Application created)
```
Comment (pasindur99) [inline app/controllers/Projects.php]: Can You remove those merge conflict messages?

File: app/controllers/Projects.php [hunk 3]
```diff
@@ -575,6 +579,7 @@ public function volunteerApplication($id) {
             
 
         ];
+<<<<<<< HEAD
```
Comment (pasindur99) [inline app/controllers/Projects.php]: Can You remove those merge conflict messages?

File: app/controllers/Projects.php [hunk 4]
```diff
@@ -660,4 +665,94 @@ public function volunteerApplication($id) {
     $this->view('projects/volunteerApplicationForm', $data);
   }
 }
+=======
+    public function volunteerApplication() {
+=======
+>>>>>>> 8374cb0 (volunteer Application created)
```
Comment (pasindur99) [inline app/controllers/Projects.php]: Can You remove those merge conflict messages?

File: app/controllers/Projects.php [hunk 5]
```diff
@@ -660,4 +665,94 @@ public function volunteerApplication($id) {
...
+              
+                header('location:' . URL_ROOT . '/pages/index');
+            }
+        }
+ 
+        $data['vol_opportunity']= $this->projectModel->getvolOpportunityID($id); 
+    
+    $this->view('projects/volunteerApplicationForm', $data);
+  }
+>>>>>>> 7596dfa (volunteer form)
```
Comment (pasindur99) [inline app/controllers/Projects.php]: Can You remove those merge conflict messages?

File: app/models/Complaint.php [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+            $this->db->bind(':subject', $data['subject'] );
+            $this->db->bind(':description', $data['description'] );
+
+            if($this->db->execute()) {
+                return true;
+            } else {
+                return false;
+            }
+        }
+    }
```
Comment (pasindur99) [inline app/models/Complaint.php]: EOL

File: app/views/users/user/inquries.php [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+</div>
+
+<div id="footer">
+            <?php
+            require APP_ROOT . '/views/components/footer.php';
+            ?>
+        </div>
+
+</body>
+</html>
```
Comment (pasindur99) [inline app/views/users/user/inquries.php]: EOL

**Review by pasindur99 — CHANGES_REQUESTED** [review]
Need some minor changes.

**Review by azeimtiaz — APPROVED** [review]
All good

**Pull request merged**